lib/ansible/modules/windows/win_dns_client.ps1

```diff
@@ -193,29 +209,29 @@ Try {
 
     Write-DebugLog ("Validating adapter name {0}" -f $adapter_names)
 
-    $adapters = @($adapter_names)
-
-    If($adapter_names -eq "*") {
-        $adapters = Get-NetAdapter | Select-Object -ExpandProperty Name
-    }
     # TODO: add support for an actual list of adapter names
     # validate network adapter names
-    ElseIf(@(Get-NetAdapter | Where-Object Name -eq $adapter_names).Count -eq 0) {
-        throw "Invalid network adapter name: {0}" -f $adapter_names
+    $adapter_names = @($adapter_names)
+
+    foreach($adapter_name in $adapter_names) {
+        If($adapter_name -eq "*") {
+            $adapters = Get-NetAdapter | Select-Object -ExpandProperty Name
+        } ElseIf(@(Get-NetAdapter | Where-Object {$_.Name -like $adapter_names}).Count -eq 0) {
```

Comment:
Why not use the `-name` parameter of `Get-NetAdapter`?

Reply:
Cause it was there before and I haven't noticed it until now.